clang/test/Sema/conversion.c

```diff
@@ -441,10 +440,11 @@ void double2float_test2(double a, float *b) {
 
 float sinf (float x);
 double double2float_test3(double a) {
-    return sinf(a); // expected-warning {{implicit conversion loses floating-point precision: 'double' to 'float'}}
+  return sinf(a); // expected-warning {{implicit conversion loses floating-point precision: 'double' to 'float'}}
 }
 
 float double2float_test4(double a, float b) {
   b -= a; // expected-warning {{implicit conversion when assigning computation result loses floating-point precision: 'double' to 'float'}}
   return b;
 }
+float f = 1.0 / 2.0; // expected-warning {{implicit conversion between floating point types of different sizes}}
```

Comment:
All of the changes in this file except for this one are unrelated whitespace changes that should be backed out.

Reply:
I fixed the unrelated whitespace changes, but clang-format seems to alter it.

Reply:
Are you running https://clang.llvm.org/docs/ClangFormat.html#script-for-patch-reformatting or clang-format directly? If you run clang-format over the file, it will adjust the whole file. If you run that patch formatter, it will format only the lines changed by the patch.